### PR TITLE
Picking metamodel changes onto master

### PR DESCRIPTION
**Pull request by imbur**

### CODE REVIEW

File: plugins/hu.bme.mit.massif.simulink.api/src/hu/bme/mit/massif/simulink/api/Exporter.java [hunk 1]
```diff
@@ -493,19 +493,18 @@ private void exportBlocks(EList<Block> sameLevelBlocks) throws SimulinkApiExcept
...
+                String propertyName = parameter.getName();
+                commandFactory.setParam().addParam(getFQN(block)).addParam(propertyName).addParam(parameter.getValue()).execute();
             }
 
         }
 
         // When there is at least one block on the same level, check that only blocks are there on the same level will be present in the exported model
         String containerFQN = getContainerFQN(sameLevelBlocks);
 
-        MatlabCommand findExistingBlocks = commandFactory.findSystem().addParam(containerFQN).addParam("IncludeCommented").addParam("on")
```
Comment (abelhegedus) [inline plugins/hu.bme.mit.massif.simulink.api/src/hu/bme/mit/massif/simulink/api/Exporter.java]: You seem to be reverting the commented out block support.

Comment (imbur) [reply plugins/hu.bme.mit.massif.simulink.api/src/hu/bme/mit/massif/simulink/api/Exporter.java]: True, this should not be this way

File: plugins/hu.bme.mit.massif.simulink.editor/src/hu/bme/mit/massif/simulink/presentation/SimulinkEditor.java [hunk 2]
```diff
@@ -1,13 +1,3 @@
-/*******************************************************************************
```
Comment (abelhegedus) [inline plugins/hu.bme.mit.massif.simulink.editor/src/hu/bme/mit/massif/simulink/presentation/SimulinkEditor.java]: Copyright header lost on many files (delete empty before regenerating)

File: releng/hu.bme.mit.massif.parent/pom.xml [hunk 3]
```diff
@@ -71,7 +71,7 @@
 		<tycho.version>1.2.0</tycho.version>
 		<xtend.compiler.version>2.13.0</xtend.compiler.version>
 		<viatra.compiler.version>2.0.0</viatra.compiler.version>
-		<viatra.repository.url>http://download.eclipse.org/viatra/updates/release/</viatra.repository.url>
```
Comment (abelhegedus) [inline releng/hu.bme.mit.massif.parent/pom.xml]: Should be release/2.0.1

File: releng/hu.bme.mit.massif.parent/pom.xml [hunk 4]
```diff
@@ -71,7 +71,7 @@
 		<tycho.version>1.2.0</tycho.version>
 		<xtend.compiler.version>2.13.0</xtend.compiler.version>
 		<viatra.compiler.version>2.0.0</viatra.compiler.version>
```
Comment (abelhegedus) [inline releng/hu.bme.mit.massif.parent/pom.xml]: Should be 2.0.1

File: plugins/hu.bme.mit.massif.simulink.api/src/hu/bme/mit/massif/simulink/api/util/BasicSimulinkEMFOperationsAPI.java [hunk 5]
```diff
@@ -1,195 +0,0 @@
...
-import java.util.Arrays;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Set;
-
-/**
- * 
- * @author Abel Hegedus
- */
-public final class BasicSimulinkEMFOperationsAPI {
```
Comment (abelhegedus) [inline plugins/hu.bme.mit.massif.simulink.api/src/hu/bme/mit/massif/simulink/api/util/BasicSimulinkEMFOperationsAPI.java]: This is not needed any more?

File: plugins/hu.bme.mit.massif.simulink.editor/src/hu/bme/mit/massif/simulink/presentation/SimulinkActionBarContributor.java [hunk 6]
```diff
@@ -1,3 +1,13 @@
...
+ * Copyright (c) 2010-2013, Embraer S.A., Budapest University of Technology and Economics
+ * All rights reserved. This program and the accompanying materials 
+ * are made available under the terms of the Eclipse Public License v1.0 
+ * which accompanies this distribution, and is available at 
+ * http://www.eclipse.org/legal/epl-v10.html 
+ *
+ * Contributors: 
+ *     Abel Hegedus, Akos Horvath - initial API and implementation 
+ *******************************************************************************/
 /**
```
Comment (abelhegedus) [inline plugins/hu.bme.mit.massif.simulink.editor/src/hu/bme/mit/massif/simulink/presentation/SimulinkActionBarContributor.java]: Remove empty headers

Comment (imbur) [reply plugins/hu.bme.mit.massif.simulink.editor/src/hu/bme/mit/massif/simulink/presentation/SimulinkActionBarContributor.java]: Should I do the same thing for all (model, edit and editor) code?

File: plugins/hu.bme.mit.massif.simulink.api/src/hu/bme/mit/massif/simulink/api/util/BasicSimulinkEMFOperationsAPI.java (hunk 5 above)
Comment (imbur) [reply plugins/hu.bme.mit.massif.simulink.api/src/hu/bme/mit/massif/simulink/api/util/BasicSimulinkEMFOperationsAPI.java]: No, this is no longer used.

**Comment by imbur** [comment]
Keeping the issue-8 branch for further MM updates in the future.

**Comment by abelhegedus** [comment]
No, please delete the branch and then upload it again with the same name based on master whenever you have new changes!

**Comment by imbur** [comment]
OK

**Pull request merged**